### PR TITLE
Feature/library report

### PR DESCRIPTION
**Pull request by igormarchenko**

### CODE REVIEW

File: dspace-jspui/src/main/webapp/static/webix/fonts/font-license.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Font Awesome
```
Comment (sergpet) [inline dspace-jspui/src/main/webapp/static/webix/fonts/font-license.txt]: what for ?

File: dspace-jspui/src/main/java/ua/edu/sumdu/essuir/controllers/ReportController.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+
+@Controller
+@RequestMapping(value = "/statistics")
+public class ReportController {
+    @Autowired
```
Comment (sergpet) [inline dspace-jspui/src/main/java/ua/edu/sumdu/essuir/controllers/ReportController.java]: please use Resource instead

File: dspace-jspui/src/main/webapp/stat/home-stat.jsp [hunk 3]
```diff
@@ -26,9 +26,8 @@
 
 <dspace:layout locbar="nolink" title="Statistics" feedData="NONE">
     <ul class="list-group">
-        <li class="list-group-item"><a href="/stat/stat.jsp">Personal</a></li>
+        <li class="list-group-item"><a href="/statistics/report">Статистика</a></li>
```
Comment (van2007) [inline dspace-jspui/src/main/webapp/stat/home-stat.jsp]: why is it in russian?

File: dspace-jspui/src/main/java/ua/edu/sumdu/essuir/controllers/ReportController.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+                for(Faculty faculty :  faculties.values()) {
+                    result.add(faculty.generateJSONbyDate(dateFrom, dateTo));
+                }
+                return result.toString();
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return null;
```
Comment (sergpet) [inline dspace-jspui/src/main/java/ua/edu/sumdu/essuir/controllers/ReportController.java]: do not return null.

File: dspace-jspui/src/main/java/ua/edu/sumdu/essuir/controllers/ReportController.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    private ReportService reportService;
+
+    @RequestMapping(value = "/person", method = RequestMethod.GET, produces = "application/json; charset=utf-8")
+    @ResponseBody
+    public String getPersonList(@RequestParam("from") String from, @RequestParam("to") String to, HttpServletRequest request) {
+        try {
+            if(AuthorizeManager.isAdmin(UIUtil.obtainContext(request))) {
+                List<Person> persons = reportService.getPersonList();
+
+                SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
```
Comment (sergpet) [inline dspace-jspui/src/main/java/ua/edu/sumdu/essuir/controllers/ReportController.java]: should be final static in top of the methods

File: dspace-jspui/src/main/java/ua/edu/sumdu/essuir/entity/Chair.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+import org.json.JSONObject;
+import org.json.simple.JSONArray;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class Chair {
+    private String chairName;
+    private List<Person> persons = new ArrayList<>();
```
Comment (sergpet) [inline dspace-jspui/src/main/java/ua/edu/sumdu/essuir/entity/Chair.java]: better to use LinkedList

File: dspace-jspui/src/main/java/ua/edu/sumdu/essuir/entity/ChairEntity.java [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+public class ChairEntity {
+    @Id
+    @Column(name = "chair_id")
+    private Integer id;
+
+    @Column(name = "chair_name")
+    private String chairName;
+
+    @OneToOne
+    @JoinColumn(name = "faculty_id")
```
Comment (sergpet) [inline dspace-jspui/src/main/java/ua/edu/sumdu/essuir/entity/ChairEntity.java:17]: :+1:

File: dspace-jspui/src/main/java/ua/edu/sumdu/essuir/service/ReportService.java [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+import ua.edu.sumdu.essuir.repository.PersonRepository;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Service
+public class ReportService {
+
+    @Autowired
```
Comment (sergpet) [inline dspace-jspui/src/main/java/ua/edu/sumdu/essuir/service/ReportService.java]: please use Resouce

**Pull request merged**